test/integration/gobierto_budgets/elaboration_test.rb

```diff
@@ -46,14 +46,14 @@ def test_menu_subsections
   end
 
   def test_metric_boxes
-    with_current_site(placed_site) do
+    with(site: placed_site, js: true) do
```

Comment:
@ferblape I did this to avoid too much nesting on tests like:

```ruby
with_google_chrome do
  with_current_site(site) do
    with_user(user) do
      with_budget_factory(factory) do
        ...
```